frontend/src/component/admin/serviceAccounts/ServiceAccounts.tsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from 'react';
+import AdminMenu from '../menu/AdminMenu';
+import AccessContext from 'contexts/AccessContext';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { ADMIN } from 'component/providers/AccessProvider/permissions';
+import { AdminAlert } from 'component/common/AdminAlert/AdminAlert';
+import { ServiceAccountsTable } from './ServiceAccountsTable/ServiceAccountsTable';
+
+export const ServiceAccounts = () => {
+    const { hasAccess } = useContext(AccessContext);
+
+    return (
+        <div>
+            <AdminMenu />
+            <ConditionallyRender
+                condition={hasAccess(ADMIN)}
+                show={<ServiceAccountsTable />}
+                elseShow={<AdminAlert />}
+            />
+        </div>
```

Comment:
I wonder if we'll need something like https://github.com/Unleash/unleash/blob/fe1c9c70bf57362a38d71b8a6171ece49d52123a/frontend/src/component/admin/auth/AuthSettings.tsx#L39-L45

But probably, not now. At least not until this goes beta :D